src/frontend/screens/Settings/ProjectSettings/YourTeam/ReviewAndInvite/index.tsx

```diff
@@ -61,12 +54,25 @@ export const ReviewAndInvite: NativeNavigationComponent<'ReviewAndInvite'> = ({
       onSuccess: () => {
         navigation.navigate('YourTeam');
       },
-      onError: () => {
-        openSheet();
-      },
     });
   }
 
+  function clearError() {
+    if (sendInviteMutation.isIdle) {
+      sendInviteMutation.reset();
+    } else {
+      requestCancelInviteMutation.reset();
+    }
+  }
```

Comment:
should be:

if (sendInviteMutation.isError) {
      sendInviteMutation.reset()
    } 
     if(requestCancelInviteMutation.isError){
          requestCancelInviteMutation.reset()
     }